src/bootstrap-table.js

```diff
@@ -1080,7 +1080,8 @@ class BootstrapTable {
             const props = key.split('.')
 
             for (let i = 0; i < props.length; i++) {
-              if (value[props[i]] !== null) {
+              // eslint-disable-next-line eqeqeq
+              if (value[props[i]] != null) {
```

Comment:
It's better to try not to modify the ESLint configuration.

```suggestion
              if (value[props[i]] === null || value[props[i]] === undefined) {
                value = null
                break
              } else {
                value = value[props[i]]
              }
```